.github/workflows/tests.yml

```diff
@@ -155,7 +155,7 @@ jobs:
         run: "composer install --no-interaction --no-progress"
 
       - name: "Tests"
-        run: "php vendor/bin/phpunit ${{ matrix.test-filter }} --group levels"
+        run: "php vendor/bin/phpunit ${{ matrix.test-filter }} --group levels --fail-on-empty-test-suite"
```

Comment:
I'd move it to phpunit.xml please - we always want to check this.

Also there are other interesting options that we might add:


```

  executionOrder="random"
  failOnWarning="true"
  failOnRisky="true"
  failOnEmptyTestSuite="true"
  beStrictAboutOutputDuringTests="true"
```

From https://backendtea.com/post/phpunit-beyond-basic-config/#tldr